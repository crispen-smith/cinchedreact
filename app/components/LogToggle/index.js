/**
 *
 * LogToggle
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const LT = styled.a`
  flex: 1;
`;

function LogToggle(props) {
  const val = props.loggedIn ? 'LOGGED IN' : 'NOT LOGGED IN';
  return <LT onClick={props.onClick}>{val}</LT>;
}

LogToggle.propTypes = {
  loggedIn: PropTypes.bool.isRequired,
  onClick: PropTypes.func.isRequired,
};

export default LogToggle;
