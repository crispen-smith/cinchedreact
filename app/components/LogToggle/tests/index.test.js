import React from 'react';
import renderer from 'react-test-renderer';

import LogToggle from '../index';

describe('<LogToggle />', () => {
  it('Renders a MegaLogoBase', () => {
    const tree = renderer.create(<LogToggle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
