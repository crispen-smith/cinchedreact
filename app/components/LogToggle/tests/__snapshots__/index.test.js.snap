// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogToggle /> Renders a MegaLogoBase 1`] = `
<div
  className="LogToggle__LT-bRttfU dlmwTd"
>
  THIS IS THE LOG TOGGLE
</div>
`;
